app/views/devise/mailer/invitation_instructions.html.erb

```diff
@@ -0,0 +1,7 @@
+<p><%= I18n.t("devise.mailer.invitation_instructions.hello", email: @resource.email) %></p>
```

Comment:
We should probably customize this with something nicer in the message, like "We're so excited to have you join us at Epicodus! To finish your enrollment and pay your tuition, click here: [link]."